CountryPicker/CountryPicker/CountryManager.swift

```diff
@@ -9,6 +9,19 @@
 import Foundation
 import UIKit
 
+
+/// Country filtering options
+public enum CountryFilterOptions {
```

Comment:
`CountryFilterOptions` look like array may be `CountryFilterOption` will be better name for enum. Any thought ?

Reply:
Yeah makes more sense with singular context.